src/editors/data/redux/index.ts

```diff
@@ -194,7 +194,7 @@ export interface EditorState {
 
 export { actions, selectors };
 
-export function initializeStore(preloadedState = undefined) {
+export function initializeStore(preloadedState = {}) {
```

Comment:
Why is `{}` a better default than `undefined` ? I thought if we left this undefined it would use the "normal" initial state, whereas if we set it, it will use an empty initial state with no keys or values. But I could be wrong.